modular_zubbers/code/modules/mob/dead/new_player/body_markings.dm

```diff
@@ -33,6 +33,11 @@
 	icon_state = "fangs"
 	affected_bodyparts = HEAD
 
+/datum/body_marking/bubber/deer
+	name = "Deer Snout (Marking)"
+	icon_state = "deer"
+	affected_bodyparts = HEAD
+
```

Comment:
Give a credit comment by the icon state here and you're all good to merge

Reply:
Sure thing boss